kitsune/customercare/forms.py

```diff
@@ -0,0 +1,75 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _lazy
+
+from zenpy.lib.api_objects import Ticket
+
+TICKET_FORM_ID = 360000417171
+
+# See docs/zendesk.md for details about getting the valid choice values for each field:
+
+PRODUCT_FIELD_ID = 360047198211
```

Comment:
This refers to VPN which has now the slug that Kitsune uses as value in zendesk. Can we remove this and use directly our slug?